### PR TITLE
Add npm-shrinkwrap.json to control dependency updates

### PR DESCRIPTION
**Pull request by pdcastro**
Resolves: #1293
Change-type: patch

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -167,7 +167,7 @@
     "mixpanel": "^0.10.1",
     "mkdirp": "^0.5.1",
     "moment": "^2.24.0",
-    "moment-duration-format": "~2.2.2",
+    "moment-duration-format": "^2.2.2",
```
Comment (thgreasi) [inline package.json]: I think that 2.3 broke the typings but for sure we now don't have any reason to not change it to a `^`.
After all, whoever tries to change the shrinkwrap will see whether it breaks the build.

**Comment by pdcastro** [comment]
@balena-ci retest

File: package.json (hunk 1 above)
Comment (pdcastro) [reply package.json]: I didn't know about the typings, but I knew about 2 issues that I raised myself in the upstream repo:
* [ReferenceError: window is not defined in v2.3.0](https://github.com/jsmreese/moment-duration-format/issues/131)
* [ReferenceError: window is not defined in v2.3.1](https://github.com/jsmreese/moment-duration-format/issues/133)

... which had broken balena CLI and required us to pin it to `~2.2.2`. Upstream claims to have fixed it but I have not tested it myself. I was careful to add `npm-shrinkwrap.json` in a separate previous commit to the unpinning, so the CLI should still use `moment-duration-format` v2.2.2 anyway.

File: .npmrc [hunk 2]
```diff
@@ -1 +0,0 @@
-package-lock=false
```
Comment (pdcastro) [inline .npmrc]: I've found that `package-lock=false` also disables `npm-shrinkwrap.json`, hence deleting this line and the resulting blank file.

**Comment by thgreasi** [comment]
It seems that we will have to cut a release for https://github.com/balena-io-modules/node-web-streams as `@balena/node-web-streams` and update the respective line in balena request (we did something like this before for [grid-styled](https://github.com/balena-io-modules/grid/blob/ebc0bc3303679841ff9d4eed0e7896ba93b37e2d/package.json#L2)).
See: https://github.com/balena-io-modules/balena-request/blob/4f33d99336a0c697d25980aa3132cd2de18bbc65/package.json#L59

**Comment by pdcastro** [comment]
> _It seems that we will have to cut a release for https://github.com/balena-io-modules/node-web-streams as @balena/node-web-streams_

@thgreasi, although I couldn't reproduce the issue on Friday as in the [Flowdock discussion](https://www.flowdock.com/app/rulemotion/resin-frontend/threads/aPVQ3rIDGMGUN7amkz8iaQRa2Bx), I think there was indeed a problem that even affected users: see issues #1331 and #1332. Please review pull request #1334 that hopefully fixes the issue.

**Pull request merged**